### PR TITLE
Initial design

### PR DESCRIPTION
**Pull request by susilaganesan**

### CODE REVIEW

File: src/app/core/register/register.component.html [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+        </div>
+        <div>
+         <input type="password"  placeholder="Password" formControlName="password">
+        </div> 
+     </form> 
+        <div>
+          <button (click)="submit()"> Sign Up </button>
+        </div>       
+    </div>
+  </div>
```
Comment (hchockarprasad) [inline src/app/core/register/register.component.html]: Indentation refactor needed.

File: src/app/core/register/register.component.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+import { FormGroup, FormBuilder, FormControl} from '@angular/forms';
+
+@Component({
+  selector: 'app-register',
+  templateUrl: 'register.component.html',
+  styleUrls: ['../core.component.scss']
+
+})
+export class RegisterComponent implements OnInit {
+    signupForm: FormGroup;
```
Comment (hchockarprasad) [inline src/app/core/register/register.component.ts]: Indentation refactor needed.

File: src/app/core/login/login.component.ts [hunk 3]
```diff
@@ -3,7 +3,9 @@ import { FormGroup, FormBuilder, FormControl} from '@angular/forms';
 
 @Component({
   selector: 'app-login',
-  templateUrl: 'login.component.html'
+  templateUrl: 'login.component.html',
+  styleUrls: ['../core.component.scss']
+
```
Comment (hchockarprasad) [inline src/app/core/login/login.component.ts]: Remove redundant blank line.

File: src/app/core/layout/layout.component.html [hunk 4]
```diff
@@ -1,5 +1,6 @@
 <div fxLayout="row" fxLayoutALign="center center" class="h-100">
-  <app-container-left></app-container-left>
-  <app-container-right fxFlex>
-  </app-container-right>
-</div>
\ No newline at end of file
+        <app-container-left></app-container-left>
```
Comment (hchockarprasad) [inline src/app/core/layout/layout.component.html]: Indentation refactor needed.

File: src/app/core/core.module.ts [hunk 5]
```diff
@@ -2,11 +2,12 @@ import { NgModule } from '@angular/core';
 import { CoreComponent } from './core.component';
 import { ReactiveFormsModule, FormsModule } from '@angular/forms';
 import { HttpClientModule } from '@angular/common/http';
-
 import { CoreRoutingModule } from './core-routing.module';
```
Comment (hchockarprasad) [inline src/app/core/core.module.ts]: Leave a blank line between external and internal imports with external on top.

File: src/app/core/core-routing.module.ts [hunk 6]
```diff
@@ -2,6 +2,8 @@ import { NgModule } from '@angular/core';
 import { Routes, RouterModule } from '@angular/router';
 
 import { LoginComponent } from './login/login.component';
+import { RegisterComponent } from './register/register.component';
+
```
Comment (hchockarprasad) [inline src/app/core/core-routing.module.ts]: Remove redundant blank line.

File: src/app/app.module.ts [hunk 7]
```diff
@@ -3,6 +3,7 @@ import { NgModule } from '@angular/core';
 
 import { AppComponent } from './app.component';
 import { CoreModule } from './core/core.module';
+// import {FlexLayoutModule} from '@angular/flex-layout';
```
Comment (hchockarprasad) [inline src/app/app.module.ts]: Remove commented import statement.

**Comment by hchockarprasad** [comment]
@susilaganesan Please try to follow these code style for the entire project :radio:

File: src/app/core/register/register.component.html (hunk 1 above)
Comment (susilaganesan) [reply src/app/core/register/register.component.html]: ok..

File: src/app/core/register/register.component.ts (hunk 2 above)
Comment (susilaganesan) [reply src/app/core/register/register.component.ts]: ok..

File: src/app/core/login/login.component.ts (hunk 3 above)
Comment (susilaganesan) [reply src/app/core/login/login.component.ts]: ok..

File: src/app/core/core.module.ts (hunk 5 above)
Comment (susilaganesan) [reply src/app/core/core.module.ts]: ok..

File: src/app/core/core-routing.module.ts (hunk 6 above)
Comment (susilaganesan) [reply src/app/core/core-routing.module.ts]: ok..

File: src/app/app.module.ts (hunk 7 above)
Comment (susilaganesan) [reply src/app/app.module.ts]: ok..

**Pull request merged**